Yini-He/Weekly-Meetings.md

```diff
@@ -1,12 +1,55 @@
 # Weekly Meetings
 
+* [30 August 2019](#date-30-august-2019)
 * [22 August 2019](#date-22-august-2019)
 * [13 August 2019](#date-13-august-2019)
 * [19 July 2019](#date-19-july-2019)
 * [07 July 2019](#date-07-july-2019)
 * [28 June 2019](#date-28-june-2019)
 * [17 June 2019](#date-17-june-2019)
 
+### Date: 30 August 2019
+
+#### Who did you help this week?
```

Comment:
Unfortunately you've deleted my comments @minnieho1115 - you can see them in a previous commit `1d433a`.

Please can you recover them? I don't mind if you copy and paste the changes back in. If you're not sure how to do that maybe ping a question in the slack channel and see if anyone is around to help.

Reply:
Dear Kirstie,I think the previous commit is 1d4338a(not 1d433a?),right?I could see your comments in my Repository.Could I update this file and make a new pull requist now? I guess you could see your comments then.

Reply:
Yes - that was a typo in my hash - 1d4338a is right.

You don't need to make a new pull request (in fact - please do _not_ make a new pull request) but you do have to fix the merge conflicts here and add back in the changes that you deleted.